avaframe/tests/test_in1Data.py

```diff
@@ -18,7 +19,7 @@ def test_getInputData(tmp_path):
     # get input data
     dirPath = os.path.dirname(__file__)
     avaName = 'avaHockeyChannel'
-    avaDir  = os.path.join(tmp_path, avaName)
+    avaDir = os.path.join(tmp_path, avaName)
```

Comment:
consider using pathlib?

Reply:
getInputData is for com1DFAOrig which still works with os... so I did not change it